SomosMasApp/SomosMasApp/App/Shared/Modules/Nosotros/ViewController/NosotrosViewController.swift

```diff
@@ -7,59 +7,47 @@
-                         NosotrosData(image: UIImage(named:"Image_5"), name: "Maria Irola", jobTitle: "Puesto 4"),
-                         NosotrosData(image: UIImage(named:"Image_4"), name: "Marita Gomez", jobTitle: "Puesto 5"),
-                         NosotrosData(image: UIImage(named:"Image_2"), name: "Miriam Diaz", jobTitle: "Puesto 6"),
-                         NosotrosData(image: UIImage(named:"Image_3"), name: "Rodrigo Fuente", jobTitle: "Puesto 7"),
-                         NosotrosData(image: UIImage(named:"Image_1"), name: "Juan Perez", jobTitle: "Puesto 2"),
-                         NosotrosData(image: UIImage(named:"Image_4"), name: "Maria Garcia", jobTitle: "Puesto 3"),
-                         NosotrosData(image: UIImage(named:"Image_5"), name: "Maria Irola", jobTitle: "Puesto 4"),
-                         NosotrosData(image: UIImage(named:"Image_4"), name: "Marita Gomez", jobTitle: "Puesto 5"),
-                         NosotrosData(image: UIImage(named:"Image_2"), name: "Miriam Diaz", jobTitle: "Puesto 6"),
-                         NosotrosData(image: UIImage(named:"Image_3"), name: "Rodrigo Fuente", jobTitle: "Puesto 7")
-                          ]
-    
     @IBOutlet var nosotrosCollectionView: UICollectionView!
+    @IBOutlet var label: UILabel!
     
+    private let service = NosotrosService()
+    private var viewModel: NosotrosViewModel?
     
-    @IBOutlet var label: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
+        self.viewModel = NosotrosViewModel(service: service, delegate: self)
+        self.viewModel?.getMembers()
         self.nosotrosCollectionView.delegate = self
         self.nosotrosCollectionView.dataSource = self
         
         label.adjustsFontSizeToFitWidth = true
        
         nosotrosCollectionView.register(UINib(nibName: "NosotrosCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "nosotrosCell")
-    
+
+    spinnerLoadingState(state: true)
     }
 
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return nosotrosData.count
+        return viewModel?.getMembersCount() ?? 0
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "nosotrosCell", for: indexPath) as? NosotrosCollectionViewCell
         
-        cell?.name.text = nosotrosData[indexPath.row].name
-        cell?.jobTitle.text = nosotrosData[indexPath.row].jobTitle
-        cell?.photo.image = nosotrosData[indexPath.row].image
+        cell?.name.text = viewModel?.getMember(at: indexPath.row).name
+        cell?.jobTitle.text = viewModel?.getMember(at: indexPath.row).description
         
+        let imagePath = viewModel?.getMember(at: indexPath.row).image
+        let imageUrl = URL(string: imagePath!)
```

Comment:
We already talk about the force unwrapped ismvery bad practice that lead to crashes, change this and never use the "!"